test/images/windows-servercore-cache/VERSION

```diff
@@ -1 +1 @@
-1.0
+1.1
```

Comment:
we should leave this one at 1.0. This image is never being published, we only need it into staging so we can use to build image image using it.

Reply:
updated, thanks for the explanation